### DISCUSSION TITLE
Unable to use with Bluetooth  Battery Monitor (paid version) 2.22.0.1.

### DISCUSSION THREAD

**Discussion by ghost** [Skulldorom/Bluetooth-Battery-Streamdeck#3] [Q&A] [answered]
Hello.

I like very much your app for stream deck but i have API ERROR, it seems not connected with API Bluetooth Battery Monitor PAID VERSION (app already installed on my pc).

I have a BT mouse connected.

Can you help ?

**Comment by Skulldorom** [comment, marked as answer]
> Hello.
> 
> I like very much your app for stream deck but i have API ERROR, it seems not connected with API Bluetooth Battery Monitor PAID VERSION (app already installed on my pc).
> 
> I have a BT mouse connected.
> 
> Can you help ?


Hi, 
Have you enabled the api?

https://www.bluetoothgoodies.com/info/battery-monitor-api/
